### ISSUE TITLE
Should we change the default hotkey to just "Space"?

### ISSUE THREAD

**Issue by GaurangTandon** [GaurangTandon/ProKeys#117]
People sometimes, when comparing our extension to [ATE](https://chrome.google.com/webstore/detail/auto-text-expander-for-go/iibninhmiggehlcdolcilmhacighjamp) or others often remark that ATE has automatic text expansion (just type snippet name and voila!) while ProKeys requires an additional hotkey.

They don't realize the obvious advantage that:

1. I don't have to prefix my snippet name with weird symbols like @ to avoid expanding normal words (as in ATE)
2. One can also change the hotkey to "Space" in ProKeys to make the snip expansion process almost semi-automatic

-----

My question is that should the default hotkey be Space? Or should we leave it as Shift+Space?
(want discussion since didn't actually think about this much deeply before when I actually added this setting)

**Comment by iWrote** [comment]
Space does effectively make the expansion automatic. But users like me have snips overlapping with commonly used words so Shift+Space provides necessary control over the expansion. 

ProKeys should **suggest** changing hotkey to Space within the settings page. But it should be Shift+Space by default.

**Comment by GaurangTandon** [comment]
> ProKeys should suggest changing hotkey to Space within the settings page. 

Yep, that'd be a useful addition to the Getting started page #149

**Comment by GaurangTandon** [comment]
We should leave this issue open until it's actually implemented.

**Comment by iWrote** [comment]
Good practice. k.
